sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/common/ChangeFeedMetaData.java

```diff
@@ -3,14 +3,12 @@
 package com.azure.cosmos.implementation.changefeed.common;
 
 import com.azure.cosmos.implementation.Utils;
-import com.azure.cosmos.util.Beta;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.core.JsonProcessingException;
 
 /**
  * Change Feed response meta data
  */
-@Beta(value = Beta.SinceVersion.V4_35_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
 public final class ChangeFeedMetaData {
```

Comment:
Is this change intentional?
I thought we agreed on not removing Beta tags for FFCF changes. 
Same for the other model classes of FFCF. 
Since FFCF will evolve - we haven't even public previewed, I would rather not remove Beta API annotation from these models.

Reply:
Implementation package - no Beta needed.